### PR TITLE
[PLAYER-4099-FIX]

### PR DESCRIPTION
**Pull request by cacel**
* Fix some code-styling.
* Timer used for retrieve info from files is instantiated when the view appears.
* Invalidate timer when the view disappears.
* Remove observes when the view is destroyed.

### CODE REVIEW

File: PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Utils/Views/OptionTableViewCell.m [hunk 1]
```diff
@@ -43,8 +43,13 @@ - (void)setOption:(PlayerSelectionOption *)option {
   
   [self updateWithState:@([[AssetPersistenceManager sharedManager] downloadStateForEmbedCode:_option.embedCode]) title:_option.title];
   
-  [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleAssetStateChanged:) name:AssetPersistenceStateChangedNotification object:nil];
-  [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleProgressChanged:) name:AssetDownloadProgressNotification object:nil];
+  [[NSNotificationCenter defaultCenter] addObserver:self
+                                           selector:@selector(handleAssetStateChanged:) name:AssetPersistenceStateChangedNotification
```
Comment (orobles-ooyala) [inline PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Utils/Views/OptionTableViewCell.m]: To keep consistency, let's move name to a new line.

File: PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Utils/Views/OptionTableViewCell.m [hunk 2]
```diff
@@ -43,8 +43,13 @@ - (void)setOption:(PlayerSelectionOption *)option {
...
+                                             object:nil];
+  
+  [[NSNotificationCenter defaultCenter] addObserver:self
+                                           selector:@selector(handleProgressChanged:)
+                                               name:AssetDownloadProgressNotification object:nil];
```
Comment (orobles-ooyala) [inline PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Utils/Views/OptionTableViewCell.m]: To keep consistency, let's move object to a new line.

**Review by brol1dev — APPROVED** [review]

**Comment by cacel** [comment]
@orobles-ooyala all the changes were done.

File: PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Utils/Views/OptionTableViewCell.m (hunk 1 above)
Comment (orobles-ooyala) [reply PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Utils/Views/OptionTableViewCell.m]: I think you missed this one.

Comment (cacel) [reply PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Utils/Views/OptionTableViewCell.m]: snip20181001_1

It's already solved. I don't know why github doesn't show the changes.

**Review by ipadilla4 — APPROVED** [review]
LGTM

**Pull request merged**